common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -298,13 +298,13 @@ impl StateMachine {
                     return Ok(Some(applied_state));
                 }
                 EntryPayload::Membership(ref mem) => {
-                    txn_sm_meta.insert(
+                    let r = txn_sm_meta.insert(
                         &LastMembership,
                         &StateMachineMetaValue::Membership(EffectiveMembership {
                             log_id: *log_id,
                             membership: mem.clone(),
                         }),
-                    )?;
+                    );
```

Comment:
What's the purpose to remove the result checking with `?`?
`insert()` may emit a StorageError that should be popped up to notify the the server to shutdown at once.